tests/fixtures/expected/blocks/directives/admonition.html

```diff
@@ -4,11 +4,9 @@
     <meta charset="utf-8" />
 </head>
 <body>
-<div class="admonition-wrapper screencast_class">
-    <div class="admonition admonition-">
-        <p class="admonition-title">Screencast</p>
-        <p>Do you prefer video tutorials? Check out the the screencasts.</p>
-    </div>
+<div class="admonition admonition- screencast_class">
```

Comment:
That looks odd. Perhaps we should make these `admonition-default` - so if you have a normal `admonition` (not a `note` or something else), we add this `-default`.

Reply:
Yes, it looks like a good compromise!

Reply:
done